justfile

```diff
@@ -0,0 +1,155 @@
+export RUSTFLAGS := "-D warnings"
+export RUSTDOCFLAGS := "-D warnings"
```

Comment:
Are these variables used for all just recipes? Is this necessary?

Reply:
Not sure... let's remove it when it starts to get in the way.